src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -7,26 +7,24 @@ description:
   The Multi-Screen Window Placement API allows you to enumerate the displays connected to your
   machine and to place windows on specific screens.
 date: 2020-09-14
-updated: 2021-02-23
+updated: 2021-06-14
```

Comment:
Do you know when the OT will be live? We could use `scheduled: true` and merge this as soon as the edit is done. (I prefer that because nobody has to remember to loop back and merge this.)

Reply:
@michaelwasserman will know.

Reply:
We are pretty confident we'll start OT2 in M93. Maybe set the date around the first M93 Beta (Thu July 29)?

Reply:
@michaelwasserman, if you ping the URL once you know it, I can then put it into the article and take care of merging if @jpmedley approves the article in the current state, with the only pending addition being the URL (and updating the publishing date to whatever date it may be).

Reply:
I have added the `updated` date that is in the far future :-) of July and the `scheduled: true` flag. Not sure if this works for already published articles, but with the `Do not merge` label we're also on the safe side.